cmd/config-sync-controllers/main.go

```diff
@@ -95,9 +96,13 @@ func main() {
 	})
 
 	syncPeriod := 10 * time.Minute
-	cacheBuilder := cache.MultiNamespacedCacheBuilder([]string{
-		*managedNamespace, controllers.OpenshiftConfigNamespace, controllers.OpenshiftManagedConfigNamespace,
-	})
+
+	opts := cache.Options{}
+	opts.Namespaces = []string{*managedNamespace, controllers.OpenshiftConfigNamespace, controllers.OpenshiftManagedConfigNamespace}
+	cacheBuilder := func(*rest.Config, cache.Options) (cache.Cache, error) {
+		return cache.New(restConfig, opts)
+	}
```

Comment:
Wonder if, now that the multi namespace stuff is included directly, if we should just embed the cache opts into the ctrl.Options directly and let it build out a default cache using these options, suspect that would be equivalent now

Reply:
I thought this at first as well, but ctrl.Options.Namespace is just a string. We would need it to be []string to work. Wdyt?